apps/core/scenes/resultDetail/ResultDetail.js

```diff
@@ -1,49 +1,57 @@
 // @flow
 
 import * as React from 'react';
-import { View, ScrollView } from 'react-native';
+import { View } from 'react-native';
 import { StyleSheet } from '@kiwicom/universal-components';
 import { withNavigation } from '@kiwicom/margarita-navigation';
-import { PassengerCard } from '@kiwicom/margarita-components';
+import { PassengerCards } from '@kiwicom/margarita-components';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
 type Props = {|
   +detailId: ?string,
 |};
 
 class ResultDetail extends React.Component<Props> {
-  handlePassengerEditPress = () => {
-    // @TODO
+  handlePassengerEditPress = id => {
+    if (id) {
+      alert(id); // @TODO
+    }
   };
 
   render() {
-    const name = 'John Doe';
-    const gender = 'male';
-    const nationality = 'Russian';
-    const dateOfBirth = '22/04/1980';
-    const id = 'DF45SV9';
-    const insurance = 'Travel Insurance Name';
-    const passengerCount = 1;
-    const bags = [
-      { count: 2, type: '40x15x30cm, 3kg' },
-      { count: 1, type: '55x20x40cm, 8kg' },
+    const passengerCards = [
+      {
+        name: 'John Doe',
+        gender: 'male',
+        nationality: 'Russian',
+        dateOfBirth: '22/04/1980',
+        id: 'DF45SV8',
+        insurance: 'Travel Insurance Name',
+        passengerCount: 1,
+        bags: [
+          { count: 2, type: '40x15x30cm, 3kg' },
+          { count: 1, type: '55x20x40cm, 8kg' },
+        ],
+      },
+      {
+        name: 'Jana Nováková',
+        gender: 'female',
+        nationality: 'Czech',
+        dateOfBirth: '22/04/1984',
+        id: 'DF45SV9',
+        insurance: 'Travel Insurance Name',
+        passengerCount: 1,
+        bags: [{ count: 1, type: '40x15x30cm, 3kg' }],
+      },
     ];
+
     return (
       <View style={styles.container}>
-        <ScrollView>
```

Comment:
I think we should keep `<ScrollView>`, because we  want whole page with header and contact details scrollable.

Reply:
👍 Done